io.github.YouROK.TorrServer.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.YouROK.TorrServer
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+finish-args:
+  - --share=network
+modules:
+  - name: TorrServer
+    buildsystem: simple
+    build-commands:
+      - install -D TorrServer /app/bin/TorrServer
+      - install -Dm644 io.github.YouROK.TorrServer.appdata.xml /app/share/metainfo/io.github.YouROK.TorrServer.metainfo.xml
+      - install -Dm644 io.github.YouROK.TorrServer.png /app/share/icons/hicolor/512x512/apps/io.github.YouROK.TorrServer.png
+      - install -Dm644 io.github.YouROK.TorrServer.desktop /app/share/applications/io.github.YouROK.TorrServer.desktop
+      - install -Dm755 run.sh /app/bin/run.sh
+
+    sources:
+      - type: file
+        url: https://github.com/YouROK/TorrServer/releases/download/MatriX.128/TorrServer-linux-amd64
```

Comment:
why not building from source and have aarch64?

Reply:
no tutorial for go apps

Reply:
there are many go apps build from source